### PR TITLE
[NUI] Fix Slider Thumb shown issue

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- After default theme update, Slider Thumb disappears when user presses it.
- The reason is that the default resource is not enough,
 and the code is just to get the empty resource. (null value)
- Fixed to show slider thumb for normal, pressed, and disabled states.

### API Changes ###
- N/A

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 1]
```diff
@@ -1460,7 +1460,19 @@ private void UpdateState(bool isFocusedNew, bool isPressedNew)
             if (!isFocused && !isPressed)
             {
                 ControlState = ControlState.Normal;
-                thumbImage.ResourceUrl = thumbImageUrlSelector?.Normal;
+
+                if (thumbImageUrlSelector != null)
+                {
+                    thumbImage.ResourceUrl = thumbImageUrlSelector?.Normal;
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/Slider.cs:1466]: Once the `thumbImage.ResourceUrlSelector` is set to `thumbImageUrlSelector` then you don't have to set image resource url manually whenever the control state is changed. There is a property to use for this case: `View.EnableControlStatePropagation`.

Please refer Button's case,
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI.Components/Controls/Button.Internal.cs#L273

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs]: Removed thumbImageUrlSelector variable and added 'EnableControlState = true' to Thumb ImageView.

**Review by rabbitfor — APPROVED** [review]
:)

**Review by bshsqa — APPROVED** [review]

**Review by hinohie — APPROVED** [review]
LGTM! Finally we use BorderlineWidth!

**Pull request merged**